authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc

```diff
@@ -24,57 +24,68 @@ By setting different values for the `credentialsMode` parameter in the `install-
 * **xref:../../authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc#cco-short-term-creds[Manual mode with short-term credentials for components]**: For some providers, you can use the CCO utility (`ccoctl`) during installation to implement short-term credentials for individual components. These credentials are created and managed outside the {product-title} cluster.
 
 .CCO mode support matrix
-[cols="<.^2,^.^1,^.^1,^.^1"]
+[cols="<.^2,^.^1,^.^1,^.^1,^.^1"]
 |====
-|Cloud provider |Mint |Passthrough |Manual
+|Cloud provider |Mint |Passthrough |Manual with long-term credentials |Manual with short-term credentials
 
 |{alibaba}
 |
 |
-|X
+|X ^[1]^
+|
 
 |Amazon Web Services (AWS)
 |X
 |X
-|X ^[1]^
+|X
+|X
 
+|Global Microsoft Azure
```

Comment:
Curious. The install docs refer to this platform as Microsoft Azure. Is Global part of an approved product name list?

Reply:
Soooooo I looked this up when trying to make clear things that are **not** ASH. It's "global", but appears at the beginning of the phrase here, so capitalized. Here is an example use from Microsoft: https://learn.microsoft.com/en-us/azure-stack/operator/compare-azure-azure-stack

To your point about approved names, I should probably figure out where to bring this up for inclusion in some resource in case anyone else also needs to clearly distinguish "Azure but not ASH".